ops/dev2/main.tf

```diff
@@ -14,14 +14,15 @@ locals {
 
 # Frontend React App
 resource "azurerm_storage_account" "app" {
-  account_replication_type  = "GRS" # Cross-regional redundancy
-  account_tier              = "Standard"
-  account_kind              = "StorageV2"
-  name                      = "simplereport${local.env}app"
-  resource_group_name       = data.azurerm_resource_group.rg.name
-  location                  = data.azurerm_resource_group.rg.location
-  enable_https_traffic_only = true
-  min_tls_version           = "TLS1_2"
+  account_replication_type        = "GRS" # Cross-regional redundancy
+  account_tier                    = "Standard"
+  account_kind                    = "StorageV2"
+  name                            = "simplereport${local.env}app"
+  resource_group_name             = data.azurerm_resource_group.rg.name
+  location                        = data.azurerm_resource_group.rg.location
+  enable_https_traffic_only       = true
+  min_tls_version                 = "TLS1_2"
+  allow_nested_items_to_be_public = false
```

Comment:
does this need `cross_tenant_replication_enabled = false` same as dev
https://github.com/CDCgov/prime-simplereport/pull/4840/files#diff-40351b4e7aab1996af023dff18fa9e1e1bccc6f745afb8b29b854ced5a367b4bR19

Reply:
Yes! Thank you for calling that out.

Reply:
fixed